docs/en/04_Changelogs/4.0.0.md

```diff
@@ -1051,6 +1052,26 @@ A very small number of methods were chosen for deprecation, and will be removed
 * `ChangeSet` and `ChangeSetItem` have been added for batch publishing of versioned dataobjects.
 * `DataObject.table_name` config can now be used to customise the database table for any record.
 * `DataObjectSchema` class added to assist with mapping between classes and tables.
+* `DBMoney` values are now treated as empty only Amount is null. Values without Currency
+  will be formatted in the default locale.
+
+The below methods have been added or had their functionality updated to `DBDate`, `DBTime` and `DBDatetime`
+* `getTimestamp()` added to get the respective date / time as unix timestamp (seconds since 1970-01-01)
+* `Format()` method now use CLDR format strings, rather than PHP format strings.
+  See http://userguide.icu-project.org/formatparse/datetime.
+* getISOFormat() added which returns the standard date/time ISO 8601 pattern in CLDR format.
+* Dates passed in m/d/y format will now raise a notice but will be parsed.
+  Dates passed to constructors should follow ISO 8601 (y-m-d).
+* 2-digit years will raise a notice.
```

Comment:
"2-digit years in `setValue()` will raise a notice, since they're ambiguous without additional formatting hints"